web/src/components/Cards/jobShowCarousel.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import  {Carousel,Card} from 'react-bootstrap'
```

Comment:
(Re this whole file:) Whitespace matters. JS may not have semantic whitespace like Python, but good and consistent formatting makes your code more maintainable.